cmd/query/app/ui/placeholder/gen_assets.go

```diff
@@ -213,7 +213,7 @@ var _escData = map[string]*_escFile{
 		name:    "index.html",
 		local:   "cmd/query/app/ui/placeholder/public/index.html",
 		size:    376,
-		modtime: 1547655513,
+		modtime: 1586972220,
```

Comment:
revert this, it's not changing in this PR.

`git checkout master cmd/query/app/ui/placeholder/gen_assets.go`

Reply:
reverted